akka-docs/src/test/java/jdocs/testkit/ParentChildTest.java

```diff
@@ -123,26 +119,21 @@ public void testingWithoutParent() {
 
   @Test
   public void testingWithCustomProps() {
-    TestProbe probe = new TestProbe(system);
-    Props childProps = Props.create(MockedChild.class);
-    TestActorRef<DependentParent> parent = TestActorRef.create(system, Props.create(DependentParent.class, childProps));
+    final TestProbe probe = new TestProbe(system);
+    final Props childProps = Props.create(MockedChild.class);
+    final ActorRef parent = system.actorOf(Props.create(DependentParent.class, childProps, probe.ref()));
 
     probe.send(parent, "pingit");
 
-    // test some parent state change
-    assertTrue(parent.underlyingActor().ponged == true || parent.underlyingActor().ponged == false);
```

Comment:
This was the faulty assert expecting async interaction between parent and child to be available through TestActorRef parent